genesyscloud/outbound_callabletimeset/data_source_genesyscloud_outbound_callabletimeset.go

```diff
@@ -0,0 +1,41 @@
+package outbound_callabletimeset
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	gcloud "terraform-provider-genesyscloud/genesyscloud"
+)
+
+/*
+   The data_source_genesyscloud_outbound_callabletimeset.go contains the data source implementation
+   for the resource.
+*/
+
+// dataSourceOutboundCallabletimesetRead retrieves by name the id in question
+func dataSourceOutboundCallabletimesetRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	sdkConfig := meta.(*gcloud.ProviderMeta).ClientConfig
+	proxy := newOutboundCallabletimesetProxy(sdkConfig)
+
+	name := d.Get("name").(string)
+
+	return gcloud.WithRetries(ctx, 15*time.Second, func() *resource.RetryError {
+		callableTimeSetId, retryable, err := proxy.getOutboundCallabletimesetIdByName(ctx, name)
+
+		if err != nil && !retryable {
+			return resource.NonRetryableError(fmt.Errorf("Error Outbound Callabletimeset %s: %s", name, err))
```

Comment:
We're using `retry.NonRetryableError()` (and `RetryableError()`)  instead now. The function in the `resource` package is deprecated